sample/NCronJobSample/TestRetryJob.cs

```diff
@@ -0,0 +1,34 @@
+using LinkDotNet.NCronJob;
+
+namespace NCronJobSample;
+
+#pragma warning disable CA1848
```

Comment:
If many of the settings are annonying we should have a separate editorconfig for the sample project.
I am not a big fan of a palette of `prgram warning disable`s

Reply:
Yeah that was me being impatient, trying to get a test job working. I can fix this or we can have a separate editorconfig as you mention.